exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/flatten/FlattenRecordBatch.java

```diff
@@ -99,6 +100,22 @@ private void clear() {
     }
   }
 
+  public enum Metric implements MetricDef {
+    NUM_INCOMING_BATCHES,
```

Comment:
Strangely, these enum names show up as labels in the web UI. Maybe `INCOMING_BATCH_COUNT`? The `AVG_` and `TOTAL_` names seem readable enough...